src/Components/Members/index.js

```diff
@@ -1,25 +1,53 @@
 import { useEffect, useState } from 'react';
 import styles from './members.module.css';
+import TableMember from './TableMember';
 
 function Members() {
   const [members, setMembers] = useState([]);
 
+  const getMembers = async () => {
+    try {
+      const reponse = await fetch(`${process.env.REACT_APP_API_URL}/member`);
+      const data = await reponse.json();
+      setMembers(data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
   useEffect(() => {
-    fetch(`https://jsonplaceholder.typicode.com/users`)
-      .then((response) => response.json())
-      .then((response) => {
-        setMembers(response);
-      });
+    getMembers();
   }, []);
 
+  const memberDelete = async (memberId) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/member/${memberId}`, {
+        method: 'DELETE'
+      });
+      console.log(response);
+      setMembers([...members.filter((member) => member._id !== memberId)]);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
   return (
     <section className={styles.container}>
-      <h2>Members</h2>
-      <div>
-        {members.map((employee) => {
-          return <div key={employee.id}>{employee.name}</div>;
-        })}
+      <div className={styles.titleContainer}>
+        <h2 className={styles.letterColour}>Members</h2>
+        <div className={styles.addContainer}>
+          <img
+            className={styles.imgSize}
+            src={`${process.env.PUBLIC_URL}/assets/images/btn-add.png`}
+          />
+          <p>Add Member</p>
+        </div>
       </div>
+      {members.length === 0 ? (
```

Comment:
can't we just validate `members.length`? Since 0 is falsy it should work in the same way